test/unit/deployment/test_language_container_deployer.py

```diff
@@ -230,16 +230,38 @@ def test_slc_deployer_get_language_definition(
     assert command == expected_command
 
 
-def mock_deployer(pyexasol_conn, pylanguage_alias, bfs_path):
-    deployer = LanguageContainerDeployer(pyexasol_conn, language_alias, bfs_path)
-    deployer.upload_container = Mock()
-    deployer.activate_container = Mock()
-    return deployer
-
```

Comment:
Not used anywhere. I suspect that it was replaced by the fixture at line 71 at some point, but not removed here.